src/DI/Container.php

```diff
@@ -107,6 +107,26 @@ public function getService($name)
 
 
 	/**
+	 * Gets the service class by name.
+	 * @param  string
+	 * @return object
+	 * @throws MissingServiceException
+	 */
+	public function getServiceClass($name)
```

Comment:
This implies it can get the class for real, which means the code that resolves the class name in prepareClassList() should be moved to (ideally) some public static function and reused.